lib/datasource/gitlab-tags/index.ts

```diff
@@ -34,9 +36,11 @@ export async function getReleases({
   const urlEncodedRepo = encodeURIComponent(repo);
 
   // tag
-  const url = URL.resolve(
+  const url = joinUrlParts(
     depHost,
-    `/api/v4/projects/${urlEncodedRepo}/repository/tags?per_page=100`
+    `api/v4/projects`,
```

Comment:
Would it fail if `/api/v4/projects`?

Reply:
nope, it doesn't matter for url-join, it simply joins the strings with `/` and removes any duplicate `/`.
it also removes any `/` at end of joind url